source/jsonnet/england-wales/ccs/blocks/individual/country_of_birth.jsonnet

```diff
@@ -32,7 +32,7 @@ local proxyTitle = {
   ],
 };
 
-{
+function(census_date) {
```

Comment:
Is this variable named correctly? Can't any date be passed into these functions? If it's fixed then maybe it should reference the static date throughout rather than pass a var.

Reply:
I agree. If we used an external variable (`--ext-str`) rather than a top level argument (`--tla-str`) then it would be available in any Jsonnet file without being passed by using `std.extVar('census_date')`.

Reply:
I have made the `census_date` external for all schemas.